calc.rb

```diff
@@ -0,0 +1,35 @@
+
+
+class Calculator
+
+	attr_accessor :num1,:num2
+
+	def initialize num1, num2
+		self.num1 = num1
+		self.num2 = num2
+	end
+
+	def add
+		 puts "#{Integer(num1)} + #{Integer(num2)} = #{Integer(num1) + Integer(num2)}"
+	end
+
+	def mul
+		puts "#{Integer(num1)} + #{Integer(num2)} = #{Integer(num1) * Integer(num2)}"
```

Comment:
You repeat the same output message for all the operations - this should be #{Integer(num1)} * #{Integer(num2)}, not #{Integer(num1)} + #{Integer(num2)}